### ISSUE TITLE
libpoppler version conflict on Ubuntu

### ISSUE THREAD

**Issue by beanumber** [ropensci/pdftools#65]
I know this is really an issue with my system and not `pdftools`, but I figure it might affect other uses. (I think this is related but different than #9). 

I have a `libpoppler` version conflict issue like [this person](https://askubuntu.com/questions/1112856/how-to-install-poppler-0-73-on-ubuntu-18-04/1123774#1123774), but I am using the `opencpu` PPA, so both versions in question are 0.74.

Does Ubuntu know that `bionic` is newer than `xenial`? I am trying to use the PPA but am getting this error:

```
The following packages have unmet dependencies:
 libpoppler-cpp0v5 : Depends: libpoppler85 (= 0.74.0-bionic0) but 0.74.0-xenial0 is to be installed
E: Unable to correct problems, you have held broken packages.
```
It says it's "to be installed", but it's already installed. 

Does it just think that `0.74.0-xenial0` is newer than `0.74.0-bionic0` because of alphabetical order? How do I fix this?

**Comment by beanumber** [comment]
```> apt-cache policy libpoppler85 
libpoppler85:
  Installed: 0.74.0-xenial0
  Candidate: 0.74.0-xenial0
  Version table:
 *** 0.74.0-xenial0 100
        100 /var/lib/dpkg/status
     0.74.0-bionic0 500
        500 http://ppa.launchpad.net/opencpu/poppler/ubuntu bionic/main amd64 Packages
```

**Comment by beanumber** [comment]
I'm guessing this is happening because I recently upgraded this system from `xenial` to `bionic` and I previously had the `xenial` version of the PPA, but now I am installing the R package anew. 

It's also probably a result of Ubuntu running out of letters in the alphabet!

So maybe it's not a widespread issue. 

I was able to install `pdftools` by removing the PPA and then installing the Ubuntu version of `libpoppler-cpp-dev`, so I'm closing this issue. Thanks for letting me think out loud!

But I still don't know how I can upgrade from 0.62 to 0.74 since:
```
The following packages have been kept back:
  libpoppler-cpp-dev libpoppler-cpp0v5 libpoppler-dev libpoppler-private-dev
```
